### ISSUE TITLE
Access time stacking

### ISSUE THREAD

**Issue by speddle** [esaracco/wopits#3]
Here is another possible feature:
Update time last click on postits in database and change stacking order to display the last clicked postit on top.

**Comment by esaracco** [comment]
@speddle We can discuss this, but I think the user wants to control the stacking order on their own.
Why not a cell menu to control it by cell (clicked, created, updated...)?

**Comment by speddle** [comment]
Yeah that would be OK. It gives me what I wanted and also supports the existing rules. I would probably do it for the whole wall, but changing one cell at a time would be OK because it is more like a setting which I would RARELY change. I seem to be the only one providing feedback. Hopefully I am not the only user :-)

**Comment by esaracco** [comment]
@speddle GitHub is primarily intended for developers.
Once wopits will be stabilized I will find another way for other users to report bugs and ask for features.
